Release Notes/511.md

```diff
@@ -72,6 +72,11 @@
   - Pull request: https://github.com/apple/swift-syntax/pull/2272
   - Migration steps: If necessary, change type annotations from the tuple to the `IncrementalParseResult` type.
 
+- `SyntaxClassification` gained a new case: `parameterLabel`
```

Comment:
The Swift grammar calls theses "argument labels" and the second name "parameter name" FWIW.

Reply:
I agree that argument label is the better name here since it’s a label in a function call where you pass arguments (opposed to the function declaration which has parameters).